Userland/Libraries/LibAudio/MP3Loader.cpp

```diff
@@ -829,7 +828,7 @@ void MP3LoaderPlugin::synthesis(double* V, double samples[32], double result[32]
     for (size_t i = 0; i < 64; i++) {
         V[i] = 0;
         for (size_t k = 0; k < 32; k++) {
-            const double N = AK::cos((16 + i) * (2 * k + 1) * AK::Pi<double> / 64.0);
+            const double N = MP3::Tables::SynthesisSubbandFilterCoefficients[i][k]; //AK::cos((16 + i) * (2 * k + 1) * AK::Pi<double> / 64.0);
```

Comment:
I think you can squash this commit, you just introduced this feature, so making a separate optimizing commit is not really useful
Same for the other quasi fixup commits

Reply:
Yep, squashed it is